code/modules/surgery/organs/organ.dm

```diff
@@ -30,7 +30,8 @@
 	var/emp_proof = FALSE //is the organ immune to EMPs?
 	var/hidden_pain = FALSE //will it skip pain messages?
 	var/requires_robotic_bodypart = FALSE
-
+	/// Is this a special (or cybernetic) organ that clings will not want to lose?
+	var/special = FALSE
```

Comment:
Absolutely terrible name. Don't. It sure is meaningful for your PR but someone else looking at the code 3 months down the road who has no context #14589 even existed would be cursing you. Vehemetly.
`var/preserve_on_cling_transform` / `var/expel_on_cling_transform` . Or something else more descriptive than just "maybe has some meaning somewhere in the code idk".

(same for `keep_special` argument to `set_species` / `create_organs`)

Reply:
Good point, although  I presumed the comment would help that, but will change it